ncbi/src/main/java/org/reactome/release/EuropePMC.java

```diff
@@ -0,0 +1,375 @@
+
+		writeEuropePMCProfileFile();
+		writeEuropePMCLinkFile(graphDBSession);
+
+		logger.info("Finished writing Europe PMC files");
+	}
+
+	/**
+	 * Writes Europe PMC profile file to pre-set output directory.  Content describes Reactome, is static, and
+	 * is pre-defined.
+	 * @throws IOException Thrown if creating or appending for file fails
+	 */
+	private void writeEuropePMCProfileFile() throws IOException {
+		Path europePMCProfileFilePath = getEuropePMCProfileFilePath();
+
+		Files.deleteIfExists(europePMCProfileFilePath);
+		Files.createFile(europePMCProfileFilePath);
+
+		logger.info("Writing Europe PMC Profile file");
+
+		appendWithNewLine(getEuropePMCProfileXML(), europePMCProfileFilePath);
+
+		logger.info("Finished writing Europe PMC Profile file");
+	}
+
+	/**
+	 * Produces Europe PMC profile XML content
+	 * @return XML string describing Reactome as a data provider to Europe PMC
+	 */
+	private String getEuropePMCProfileXML() {
+		return String.join(System.lineSeparator(),
+			getXMLDeclaration(),
+			"<providers>",
+			indentString("<provider>", 1),
+			indentString("<id>" + reactomeProviderID + "</id>",2),
+			indentString("<resourceName>Reactome</resourceName>", 2),
+			indentString(
+				"<description>" +
+					"Reactome is a free, open-source, curated and peer-reviewed pathway database. " +
+					"Our goal is to provide intuitive bioinformatics tools for the visualization, " +
+					"interpretation and analysis of pathway knowledge to support basic research, genome analysis, " +
+					"modeling, systems biology and education." +
+				"</description>",
+				2
+			),
+			indentString("<email>help@reactome.org</email>", 2),
+			indentString("</provider>", 1),
+			"</providers>"
+		);
+	}
+
+	/**
+	 * Writes Europe PMC link file to pre-set output directory.  Pathway and literature reference data retrieved from
+	 * the graph database.
+	 * @param graphDBSession Neo4J Driver Session object for querying the graph database
+	 * @throws IOException Thrown if creating or appending for either file fails
+	 */
+	private void writeEuropePMCLinkFile(Session graphDBSession) throws IOException {
+		Path europePMCLinkFilePath = getEuropePMCLinkFilePath();
+
+		Files.deleteIfExists(europePMCLinkFilePath);
+		Files.createFile(europePMCLinkFilePath);
+
+		logger.info("Writing Europe PMC Link file");
+
+		appendWithNewLine(getXMLDeclaration(), europePMCLinkFilePath);
+		appendWithNewLine(getOpenRootTag(), europePMCLinkFilePath);
+
+		for (EuropePMCLink europePMCLink : fetchEuropePMCLinks(graphDBSession)) {
+			appendWithNewLine(europePMCLink.getLinkXML(), europePMCLinkFilePath);
+		}
+		appendWithNewLine(getCloseRootTag(), europePMCLinkFilePath);
+
+		logger.info("Finished writing Europe PMC Link file");
+	}
+
+	/**
+	 * Standard XML declaration header
+	 * @return XML declaration as String
+	 */
+	private String getXMLDeclaration() {
+		return "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>";
```

Comment:
Does this need to be a function, or could it be String constant?

Reply:
I've re-worked how the XML is generated, so this is produced programmatically.  I'll need you guys to look over the new XML generation logic when all changes are done and the PR is ready for the second round of review.  I will let you know when to look it over.

Reply:
Ok.